Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM calthorpeanalytics/python3-geo:3.6.3-1.0.0
```

Comment:
Will need to update this to use a publicly accessible image (!). Basically, same as the one I created and submitted here: https://github.com/UDST/urbanaccess/pull/18